.circleci/config.yml

```diff
@@ -251,7 +251,7 @@ jobs:
           name: Install Python package
           command: |
             source env/bin/activate
-            DEV_RELEASE=1 \
+            MACOSX_DEPLOYMENT_TARGET="" DEV_RELEASE=1 \
```

Comment:
Builds the module to generate stubs using the local target so we can actually load it.